generator/generator.go

```diff
@@ -228,8 +248,8 @@ func main() {
 		}
 	}
 
-	if *outputFile != "" && *outputDir != "" {
-		log.Exitf("Error: cannot specify both outputFile (%s) and outputDir (%s)", *outputFile, *outputDir)
+	if *ocStructsOutputFile != "" && *ocStructsOutputDir != "" {
+		log.Exitf("Error: cannot specify both ocStructsOutputFile (%s) and ocStructsOutputDir (%s)", *ocStructsOutputFile, *ocStructsOutputDir)
 	}
```

Comment:
Can we add error handling for the case that the user gives us a path to the generated structs, but we're also generating them?

Reply:
I've also added error handling for the opposite case: requiring users to supply the path to the generated structs when only path structs are being generated.